samples/subsys/usb/webusb/src/webusb_serial.c

```diff
@@ -148,13 +148,13 @@ struct dev_common_descriptor {
 		struct usb_product_descriptor {
 			u8_t bLength;
 			u8_t bDescriptorType;
-			u8_t bString[0x0E - 1];
+			u8_t bString[0x0E - 2];
 		} __packed unicode_product;
 
 		struct usb_sn_descriptor {
 			u8_t bLength;
 			u8_t bDescriptorType;
-			u8_t bString[0x0C - 1];
+			u8_t bString[0x0C - 2];
 		} __packed unicode_sn;
```

Comment:
No this is correct here and match the spec which says, error comes from length values.
"The UNICODE string descriptor (shown in Table 9-16) is not NULL-terminated. The string length is
computed by subtracting two from the value of the first byte of the descriptor."

Think we need to fix all this stuff in a proper way.

Reply:
Oops my bad your patch looks good.